scripts/python/singlepss.py

```diff
@@ -608,80 +653,75 @@ def pss_sighandler_load(data, sig, sigdata):
-				src = src[:len(src)-1]
+			if ord(src[len(src) - 1]) == 0x0a:
+				src = src[:len(src) - 1]
 
 			# add it to the map and report
-			try: 
+			try:
 				nicks[key] = pss.PssContact(nick, key, addr, src)
 				# \todo function to strip 0x or store no 0x only add on send
 				remotekeys[nick] = key[2:]
 
 			# \todo delete the record from the contact store
 			except:
-				wOut(PSS_BUFPFX_ERROR, [], "!!!", "stored contact '" + nick + "' has invalid data, skipping")
+				wOut(
+				    PSS_BUFPFX_ERROR, [], "!!!",
+				    "stored contact '" + nick + "' has invalid data, skipping")
 				continue
 
-			wOut(PSS_BUFPFX_INFO, [], "+++", "pss contact loaded from db '" + nick + "' (0x" + key[2:10] + ")")
+			wOut(
+			    PSS_BUFPFX_INFO, [], "+++", "pss contact loaded from db '" +
+			    nick + "' (0x" + key[2:10] + ")")
 
 		f.close()
 	except IOError as e:
-		wOut(PSS_BUFPFX_WARN, [], "!!!", "could not open contact store " + scriptPath + "/.pss-contacts: " + repr(e))
-	
+		wOut(
+		    PSS_BUFPFX_WARN, [], "!!!", "could not open contact store " +
+		    scriptPath + "/.pss-contacts: " + repr(e))
+
 	# open file for append (and create if not exist)
 	# \todo postfix db name with username
 	storeFile = open(scriptPath + "/.pss-contacts", "a", 0600)
 
 	# debug output confirming receive signal
-	wOut(PSS_BUFPFX_DEBUG, [], "", "(" + repr(sig) + ") using scriptpath " + scriptPath)
+	wOut(PSS_BUFPFX_DEBUG, [], "",
+	     "(" + repr(sig) + ") using scriptpath " + scriptPath)
 
 	# signal is not needed anymore now, unhook and stop it from propagating
 	weechat.unhook(loadSigHook)
 	return weechat.WEECHAT_RC_OK_EAT
 
 
-
 # unload cleanly
 def pss_sighandler_unload(data, sig, sigdata):
 	global storeFile
 
 	if not os.path.basename(sigdata) == "pss-single.py":
-		return weechat.WEECHAT_RC_OK	
+		return weechat.WEECHAT_RC_OK
 
 	storeFile.close()
 	return weechat.WEECHAT_RC_OK_EAT
 
 
-
 # set up hooks and commands
 
-loadSigHook = weechat.hook_signal(
-	"python_script_loaded",
-	"pss_sighandler_load",
-	""
-)
+loadSigHook = weechat.hook_signal("python_script_loaded",
+                                  "pss_sighandler_load", "")
 
-unloadSigHook = weechat.hook_signal(
-	"python_script_unloaded",
-	"pss_sighandler_unload",
-	""
-)
+unloadSigHook = weechat.hook_signal("python_script_unloaded",
+                                    "pss_sighandler_unload", "")
 
 cmd_main = weechat.hook_command(
-	"pss",
-	"connect to pss node and manage local contact list",
-	"THIS HELP OUTPUT IS STALE AND INSUFFICENT. LOOK AT THE README INSTEAD",
-	"\n"
-	"<cmd|name> [<arguments>]"
-	" <name> connect: connect to node\n"
-	" <name>     add: add new contact\n"
-	" <name>     msg: send message to contact\n"
-	" <name>    stop: disconnect from node\n"
-	" <name>     key: display public key\n"
-	" <name>    addr: display overlay address\n"
-	"\n",
-	"(if this displays anywhere please tell me)",
-	"pss_handle",
-	"")
+    "pss", "connect to pss node and manage local contact list",
```

Comment:
No one per line please. We can build the multiline help string above, that may make it clearer.